libs/ui/src/components/MediaItem/MediaItem.story.vue

```diff
@@ -1,54 +1,63 @@
 <template>
   <Story title="MediaItem">
     <Variant title="image">
-      <MediaItem
+      <ImageMedia
         src="https://image-beta.w.kodadot.xyz/ipfs/bafybeig6clbsbabxkku3m4furbj4klayyx7ovwtcocmgwqmv4nxllnbxxy" />
     </Variant>
 
     <Variant title="video">
-      <MediaItem
+      <VideoMedia
         src="https://image-beta.w.kodadot.xyz/ipfs/bafybeib22hxgpef7lpoltxsi2p2bm7jjhnntwedeupdjcz4pxgfps6dz7i"
         animation-src="https://image-beta.w.kodadot.xyz/ipfs/bafybeigy4tyylo7hvk5qovlqwahl5dnh56wzdt3rrawceyqrlmvmbw7sey"
         mime-type="video/mp4" />
     </Variant>
 
     <Variant title="audio">
-      <MediaItem
+      <NeoAudioPlayer
         src="https://image-beta.w.kodadot.xyz/ipfs/bafybeig6dnjacg4sipst4tkxm36ykkfuqkkb5dvfaale4u65hjtdlspvri"
         animation-src="https://image-beta.w.kodadot.xyz/ipfs/bafybeibuet6ba3xquuhiurq2on2i5ccibuh3s7aiqp6c2gqlokgtgwvu5y"
         mime-type="audio/mpeg" />
     </Variant>
 
     <Variant title="3d">
-      <MediaItem
+      <ModelMedia
         animation-src="https://image-beta.w.kodadot.xyz/ipfs/bafybeiemgzrougue4cwbf3g5ejnmrjrjxdy45522zecz6eukcpwg42z2g4" />
     </Variant>
 
     <Variant title="json">
-      <MediaItem
+      <JsonMedia
         src="https://image-beta.w.kodadot.xyz/ipfs/bafkreiatmzjr4tx46tyjh63bpyici6sb54cllvuozj2t2ct2vbx4ez4puy"
         mime-type="application/json" />
     </Variant>
 
     <!-- cors origin -->
     <!-- <Variant title="iframe">
-      <MediaItem src="https://kodadot.xyz/" mime-type="text/html" />
+      <IFrameMedia src="https://kodadot.xyz/" mime-type="text/html" />
     </Variant> -->
 
     <Variant title="pdf">
-      <MediaItem
+      <ObjectMedia
         src="https://image-beta.w.kodadot.xyz/ipfs/bafybeih6yrucucyamosb2nk4qch4cg4bxdv3yheortwawttslvt7ameiiu"
         mime-type="application/pdf" />
     </Variant>
 
     <Variant title="unknown">
-      <MediaItem
+      <UnknownMedia
         src="https://image-beta.w.kodadot.xyz/ipfs/bafybeih6yrucucyamosb2nk4qch4cg4bxdv3yheortwawttslvt7ameiiu"
         mime-type="unknown" />
     </Variant>
   </Story>
 </template>
 
 <script lang="ts" setup>
-import MediaItem from './MediaItem.vue'
+import { NeoAudioPlayer } from '@kodadot1/brick'
+// import AudioMedia from './type/AudioMedia.vue'
+// import IFrameMedia from './type/IFrameMedia.vue'
```

Comment:
```suggestion

```

Remove the comment or we coulda add the template for `AudioMedia` and `IFrameMedia`.

Reply:
- `AudioMedia` doesn't work since it need composable from kodadot folder..
- `IFrameMedia` have cors issue with content

Reply:
removed